maintainers/maintainer-list.nix

```diff
@@ -7103,6 +7103,18 @@
     githubId = 2025623;
     name = "Luc Chabassier";
   };
+  dwoffinden = {
+    email = "daw@hey.com";
+    github = "dwoffinden";
+    githubId = 1432131;
+    name = "Daniel Woffinden";
+    keys = [
+      {
+        longkeyid = "rsa4096/0xF3EA503B360FBD40";
```

Comment:
```suggestion
```
That does not exist to my knowledge

Reply:
Removed, thanks. Serves me right for reusing an old commit :sweat_smile: 

Seems it was removed in https://github.com/NixOS/nixpkgs/commit/c8cebff38b598a84466c387f694133585bba1a85